src/CMakeLists.txt

```diff
@@ -85,7 +84,8 @@ elseif(MSVC OR MSYS OR MINGW)
       -D_CRT_SECURE_NO_WARNINGS
       -DNDEBUG
       -D_VARIADIC_MAX=10
-      -DNOMINMAX)
+      -DNOMINMAX
+      -DCAPNP_LITE=1)
```

Comment:
CAPNP_DEFINITIONS changed above could be used with the new EXTERNAL_TOOLS flag. But current recommendation is to only use CAPNP_LITE. If 0.5.3 is adopted across all OSs, then as we've seen it already includes CAPNP_LITE define in capnp/common.h https://github.com/sandstorm-io/capnproto/blob/68bf6c59629b5bae76a01941ddf99b0a18206ae3/c%2B%2B/src/capnp/common.h#L51